### PR TITLE
phpmailer.lang-mg

### PR DESCRIPTION
**Pull request by octavebenil**
Une traduction en Malgache

### CODE REVIEW

**Comment by Senasum** [comment]
hello octavebenil !! are you French?

**Comment by Senasum** [comment]
Salut octavebenil !! Es-tu français ?

File: language/phpmailer.lang-mg.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+* PHPMailer language file: refer to English translation for definitive list
+* French Version
```
Comment (glensc) [inline language/phpmailer.lang-mg.php]: i'm sure this file is not for french

Comment (PatrickTrempond) [inline language/phpmailer.lang-mg.php]: No, it's not French, it's Malgache, language spoken in Madagascar and other islands in the same region..

Comment (glensc) [inline language/phpmailer.lang-mg.php]: yes. it was my hint for @octavebenil to fix this line as well :)

Comment (Benjamin0000) [inline language/phpmailer.lang-mg.php]: good

Comment (octavebenil) [inline language/phpmailer.lang-mg.php]: Yes, It's not a french, it's language of my country.

Comment (Synchro) [inline language/phpmailer.lang-mg.php]: Hey @octavebenil, it's great you're still around - any chance you can finish this off and then I'll merge it.

Comment (dextel2) [reply language/phpmailer.lang-mg.php]: @Synchro Can't we put Google translations for the missing ones till someone actually translate it?

Comment (Synchro) [reply language/phpmailer.lang-mg.php]: Sure - apparently nobody is particularly bothered about this translation, so you could fork it and resubmit the PR.

**Comment by Hackinet** [comment]
This was opened 4 years ago lol

**Comment by Synchro** [comment]
@hackinet true, and yet you’ve done nothing to help close it.

**Comment by Synchro** [comment]
Closing in favour of #1555

**Pull request closed without merging**